Vagrantfile

```diff
@@ -58,20 +58,27 @@ Vagrant.configure("2") do |config|
         config.vagrant.plugins = required_plugins
     end
 
-	config.vm.synced_folder ".", "/vagrant", disabled: false, type: 'rsync'
+    config.vm.synced_folder '.', configuration['base_folder'], disabled: false, type: 'rsync'
+    config.vm.provision 'file', source: 'configuration.yml', destination: configuration['base_folder'] + '/web_s2i/configuration.yml'
 
   config.vm.provision "docker" do |d|
-    d.build_image "/vagrant/docker",
-      args: "-t devopsloft/devopsloft"
+    d.post_install_provision "shell", inline:"docker network create devopsloft_network"
+    d.build_image configuration['base_folder'] + '/db_s2i',
+      args: '-t ' + configuration['namespace'] + '/' + configuration['dockerhub_db'] + ' --build-arg MYSQL_ROOT_PASSWORD=' + configuration['mysql_root_password'] + ' --build-arg MYSQL_DATABASE=' + configuration['mysql_db']
```

Comment:
The `MYSQL_ROOT_PASSWORD` is meaningless in the build context, the DB itself is not created here, same for `MYSQL_DATABASE`

Reply:
fixed